### DISCUSSION TITLE
Welcome to sc2-public-code-examples Discussions!

### DISCUSSION THREAD

**Discussion by codetricity** [codetricity/sc2-public-code-examples#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and supplement our 
[forum](https://community.theta360.guide/). 

You can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jcasman** [comment]
Hi, this is Jesse Casman, I'm a co-organizer of the theta360.guide community. I've been testing SC2 API commands. I'm interested in testing presets on the SC2 for Business. Room, Face, Lens-by-Lens. Unfortunately, I only have an SC2. Does anyone have an SC2B that can be used for testing?

**Reply by codetricity** [reply]
I have an SC2B. I don't think it has presets for Lens-by-Lens.  I believe that is a feature of the SC2.  You can use the HDR _filter of the SC2 to compare Lens-by-Lens to HDR.  I believe that the room mode of the SC2B is the same as the HDR _filter of the SC2.
